runtime/lib/backend/openqasm/OpenQasmDevice.hpp

```diff
@@ -74,9 +74,9 @@ class OpenQasmDevice final : public Catalyst::Runtime::QuantumDevice {
         const std::string &kwargs = "{device_type : braket.local.qubit, backend : default}")
     {
         device_kwargs = Catalyst::Runtime::parse_kwargs(kwargs);
-        device_shots = device_kwargs.contains("shots")
-                           ? static_cast<size_t>(std::stoll(device_kwargs["shots"]))
-                           : 0;
+        // device_shots = device_kwargs.contains("shots")
```

Comment:
remove the same `device_shots` parsing in lightning repo for the lightning devices

Reply:
https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/core/src/simulators/lightning_qubit/catalyst/LightningSimulator.hpp#L95